android/app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,15 @@
         android:name="android.hardware.camera.autofocus"
         android:required="false" />
 
+    <!-- Usage of @react-native-community/netinfo, rn-fetch-blob and Flipper
+         adds the permission requirement for ACCESS_WIFI_STATE in the final 
+         AndroidManifest of the app which makes wifi hardware a compulsary 
+         requirement to install the app. The following statement makes the 
```

Comment:
nit: no trailing whitespace

This is one bit of linting that's cross-language enough that Git itself highlights it when you use `git diff`:
![image](https://user-images.githubusercontent.com/28173/119439866-17039280-bcd8-11eb-8ed5-6b3f4bad3c0f.png)

(At least it does if you have `git config --global color.ui auto`, which I definitely recommend. See in general [my gitconfig](https://gist.github.com/gnprice/5f71b63464a76bd7e0480f207b1bbff3).)

Reply:
Thanks for the `.gitconfig`.